src/main/java/mate/academy/spring/dao/impl/UserDaoImpl.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.spring.dao.impl;
+
+import java.util.List;
+import mate.academy.spring.dao.UserDao;
+import mate.academy.spring.lib.DataProcessingException;
+import mate.academy.spring.model.User;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class UserDaoImpl implements UserDao {
+    private final SessionFactory sessionFactory;
```

Comment:
let's keep the code consistent, why some autowired fields are final and some of them are not? 
```suggestion
    private SessionFactory sessionFactory;
```